pkg/services/sqlstore/stats.go

```diff
@@ -192,23 +184,118 @@ func GetSystemUserCountStats(ctx context.Context, query *models.GetSystemUserCou
 	})
 }
 
-func GetActiveUserStats(query *models.GetActiveUserStatsQuery) error {
-	activeUserDeadlineDate := time.Now().Add(-activeUserTimeLimit)
-	sb := &SqlBuilder{}
+func GetUserStats(query *models.GetUserStatsQuery) error {
+	err := updateUserRoleCountsIfNecessary(query.MustUpdate)
```

Comment:
I wonder if there is a use case that the  `GetAdminStats` has to force an update similar to what happens here.

Reply:
That's a good question! Maybe?